ztp/policygenerator/policyGen/policyBuilder_test.go

```diff
@@ -1150,3 +1150,227 @@ spec:
 
 	assert.Equal(t, defaultComplianceType, objects[0].ComplianceType)
 }
+
+func TestNamespaceEvaluationIntervalDefault(t *testing.T) {
+	input := `
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "test"
+  namespace: "test"
+spec:
+  bindingRules:
+    justfortest: "true"
+  sourceFiles:
+    # Create operators policies that will be installed in all clusters
+    - fileName: GenericNamespace.yaml
+      policyName: "gen-sub-policy"
+    - fileName: GenericSubscription.yaml
+      policyName: "gen-sub-policy"
+    - fileName: GenericOperatorGroup.yaml
+      policyName: "gen-sub-policy"
+`
+	// Read in the test PGT
+	pgt := utils.PolicyGenTemplate{}
+	_ = yaml.Unmarshal([]byte(input), &pgt)
+
+	// Set up the files handler to pick up local source-crs and skip any output
+	fHandler := utils.NewFilesHandler("./testData/GenericSourceFiles", "/dev/null", "/dev/null")
+
+	// Run the PGT through the generator
+	pBuilder := NewPolicyBuilder(fHandler)
+	policies, err := pBuilder.Build(pgt)
+
+	// Validate the run
+	assert.Nil(t, err)
+	assert.NotNil(t, policies)
+
+	assert.Contains(t, policies, "test/test-gen-sub-policy")
+	policy := policies["test/test-gen-sub-policy"].(utils.AcmPolicy)
+	assert.Equal(t, policy.Spec.PolicyTemplates[0].ObjDef.Spec.EvaluationInterval.Compliant,
+		utils.DefaultCompliantEvaluationInterval)
+	assert.Equal(t, policy.Spec.PolicyTemplates[0].ObjDef.Spec.EvaluationInterval.NonCompliant,
+		utils.DefaultNonCompliantEvaluationInterval)
+}
+
+func TestNamespaceEvaluationIntervalOverride(t *testing.T) {
```

Comment:
I think you might be able to combine these two unit tests `TestNamespaceEvaluationIntervalDefault` and `TestNamespaceEvaluationIntervalOverride` into a single test?

Reply:
Yes, the two unit tests have been combined.